pylot/perception/detection/obstacle.py

```diff
@@ -29,12 +29,13 @@ class Obstacle(object):
             the obstacle.
         forward_speed (:obj:`float`): Forward speed of the obstacle (in m/s).
     """
-    def __init__(self, id, label, transform, bounding_box, forward_speed):
+    def __init__(self, id, label, transform, bounding_box, forward_speed, detailed_label=None):
```

Comment:
Can you set the default value of detailed_label to empty string? In this way the bounding_box logger doesn't fail if detailed_label is not set.